content/administration/upgrade.rst

```diff
@@ -1,264 +1,340 @@
+   - Making an online order (Website apps) from the product selection in your shop until the
+     checkout process and checking if everything behaves as in your production database.
 
-If you encounter issues or problems in the **production database**, please get in touch with **Odoo
-Support**:
+   This list is **not** exhaustive. Extend the example to your other apps based on your use of Odoo.
 
-#. Connect to our `Odoo Support page <https://www.odoo.com/help>`_.
-#. Under the *Ticket Description* section, select the appropriate type related to your issue but
-   **do not select** the option *An issue related to my upgrade*.
+If you face an issue while testing your upgraded test database, you can request the assistance of
+Odoo via the `support page <https://odoo.com/help>`_. In any case, it is essential to report any
+problem encountered during the testing to fix it before upgrading your production database.
 
-   .. note::
-      After upgrading to production, the support will be provided by the Support team instead of the
-      Upgrade team.
+You might encounter significant differences with standard views, features, fields, and models during
+testing. Those changes cannot be reverted on a case-by-case basis. However, if a change introduced
+by a new version breaks a customization, it is the responsibility of the maintainer of your custom
+module to make it compatible with the new version of Odoo.
 
-#. Please provide as much detail as you can (i.e., videos and screenshots to illustrate your issue).
-   This will avoid clarifying questions and speed up the resolution process significantly.
+.. tip::
+   Do not forget to test:
 
-   .. warning::
-      If you choose *An issue related to my upgrade* as ticket type, the request will be redirected
-      to another team than the support one and will slow down the processing and response time.
+   - Integrations with external software (EDI, APIs, etc.)
+   - Workflows between different apps (online sales with eCommerce, converting a lead all the way to
+     a sales order, delivery of products, etc.)
+   - Data exports
+   - Automated actions
+   - Server actions in the action menu on form views, as well as by selecting multiple records on
+     list views
 
-.. _upgrade/assistance:
+Upgrading the production database
+---------------------------------
 
-Help
-====
+Once the :ref:`tests <upgrade/test_your_db>` are completed and you are confident that the upgraded
+database can be used as your main database without any issue, it is time to plan the go-live day. It
+can be planned in coordination with Odoo's upgrade support analysts, reachable via the `support page
+<https://odoo.com/help>`_.
 
-.. _upgrade/contact:
+.. important::
+   - Any modification to your production database will **not** be saved during the upgrade of your
+     production database. This is why we recommend not using it during that time.
```

Comment:
You're right. I think we could even remove the whole sentence about modifications not saved ? Since even for On-Premise, it is logical that once the dump has been uploaded, any modification done after the upload will not be taken into account.

What do you think ? Should I simply move it to On-Premise or remove it entirely ?

Reply:
It is logical for us, but maybe not for all the customers
I would move it into the On-Premise tab just in case some customer's are not aware